src/ForkTsCheckerWebpackPluginConfiguration.ts

```diff
@@ -32,6 +33,7 @@ function createForkTsCheckerWebpackPluginConfiguration(
     issue: createIssueConfiguration(compiler, options.issue),
     formatter: createFormatterConfiguration(options.formatter),
     logger: createLoggerConfiguration(compiler, options.logger),
+    enableOverlay: options.enableOverlay !== false,
```

Comment:
You can probably just cast to a Boolean, to guarantee a boolean value is set? And wouldn't need to check specifically check for false and just take the value as-is (with casting)?

```suggestion
    enableOverlay: Boolean(options.enableOverlay),
```

Reply:
I think it's intentional - by default (for undefined) it will be true :)